wallet/src/Pos/Aeson/ClientTypes.hs

```diff
@@ -73,3 +74,15 @@ instance ToJSON ClientInfo where
             , "cabalVersion" .= showVersion ciCabalVersion
             , "apiVersion" .= ciApiVersion
             ]
+
+instance FromJSON NewBatchPayment where
```

Comment:
Did you try to generate it automatically, trying to adjust generating `Options` for your exact case?

Reply:
I haven't. Not sure could I trigger with `Options` that json `[ {address: "...", amount: ... } ]` deserializes into list of tuples. I don't think this modification is available through `Options`. Note that this is needed just to satisfy integration with the exchange and later we will deprecate it.

Reply:
Ah, those are tuples in the list, right.
I don't believe anyone will concern himself rewriting this instance when switching to new api )

Actually I just realized why this way is ok, so ok